kafka-ui-react-app/src/components/KsqlDb/__test__/KsqlDb.spec.tsx

```diff
@@ -14,7 +14,8 @@ describe('KsqlDb Component', () => {
     );
 
     it('matches snapshot', () => {
-      expect(mount(setupComponent())).toMatchSnapshot();
+      const { container } = render(setupComponent());
+      expect(container).toBeInTheDocument();
```

Comment:
here as well maybe check some other thing that the actually component renders

Reply:
discussed and done